packages/firestore/rollup.config.rn.js

```diff
@@ -15,20 +15,48 @@
  * limitations under the License.
  */
 
-import pkg from './package.json';
-
+const alias = require('@rollup/plugin-alias');
 const util = require('./rollup.shared');
 
-export default {
-  input: 'index.rn.ts',
-  output: {
-    file: pkg['react-native'],
-    format: 'es',
-    sourcemap: true
+export default [
+  {
+    input: 'export.ts',
+    output: {
+      file: 'dist/prebuild.rn.js',
+      format: 'es',
+      sourcemap: true
+    },
+    plugins: util.es2017Plugins('rn', /* mangled= */ true),
+    external: util.resolveBrowserExterns,
+    treeshake: {
+      moduleSideEffects: false
+    }
   },
-  plugins: util.es2017Plugins('rn', /* mangled= */ true),
-  external: util.resolveBrowserExterns,
-  treeshake: {
-    moduleSideEffects: false
+  {
+    input: {
+      index: 'index.ts',
+      memory: 'index.memory.ts',
+      bundle: 'index.bundle.ts'
+    },
+    output: {
+      dir: 'dist/rn',
+      format: 'es',
+      sourcemap: true
+    },
+    plugins: [
+      alias({
+        entries: [
+          {
+            find: /^(.*)\/export$/,
+            replacement: `$1\/dist/prebuild.rn.js`
+          }
+        ]
+      }),
+      ...util.es2017Plugins('rn', /* mangled= */ false)
```

Comment:
This should be mangled.

Reply:
Same as browser build, this should be false.